include/swift/Frontend/FrontendOptions.h

```diff
@@ -143,6 +143,9 @@ class FrontendOptions {
   /// debugger to use.
   bool AlwaysSerializeDebuggingOptions = false;
 
+  /// If set, inserts instrumentation useful for testing the debugger.
+  bool DebuggerTestingTransform = false;
```

Comment:
Should we call it DebuggerTestingInstrumentation?

Reply:
I'd rather keep the current name as it's shorter.